builds/opendingux/gcw-zero/run.sh

```diff
@@ -1,11 +1,25 @@
 #!/bin/sh
-cd `dirname $0`
+SOURCE="$1"
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
```

Comment:
Ah okay, was just confused why you forced "$HOME/.easyrpg" as project dir before and now not anymore but this explains it.